### DISCUSSION TITLE
resolve path: finding mdx files inside a dynamic route

### DISCUSSION THREAD

**Discussion by jonoise** [vercel/next.js#59921] [App Router]
hi guys. i have a blog and previously we were getting all static mdx files (in production) from the blog like this:

```ts
import glob from 'fast-glob'
let pages = await glob('**/*.mdx', { cwd: 'app/home/blog' })
```

useful to automatically fetch all posts and arrange them in sidebar and inside `/blog` page.

recently we added i18n. so we wrapped the whole app into a `[locale]` folder, so we added that folder to our path.

```ts
  let pages = await glob('**/*.mdx', { cwd: 'app/[locale]/home/blog' })
```

it finds the files in development, but not in production. i tried adding `path.resolve()` and `process.cwd()`, but none of them resolve the path to my mdx files. 

the full path in production looks like this `'/var/task/app/[locale]/home/blog'`

**Comment by kelerer1** [comment]
I have run into an identical issue. What was the resolution for you?

**Reply by BenLyddane** [reply]
same issue here, can't find a solution...

**Comment by zjy365** [comment]
same issue here...
